Usage.md

```diff
@@ -16,6 +16,22 @@ The app will start with subscriptions defined in `resources/subscription-example
 
 When the app has started, an admin-gui will be available at http://localhost:8012/anshar/stats 
 
+# Using docker
+
+Build the image :
+
+```shell
+docker build . -t anshar
+```
+
+Run the image :
+
+```shell
+# With default coniguration
+docker run -p 8012:8012
```

Comment:
The docker-commands are incomplete:
```
>  docker run -p 8012:8012                                                                                                                                                                                                       
"docker run" requires at least 1 argument.
See 'docker run --help'.
```

Reply:
Oops, I forgot to put the container name. Fixed it.